recipes/barrnap/0.2/meta.yaml

```diff
@@ -0,0 +1,22 @@
+about:
+  home: https://github.com/tseemann/barrnap
+  license: GPL3
+  summary: 'Barrnap predicts the location of 5S, 16S and 23S ribosomal RNA genes in
+    Bacterial genome sequ It takes FASTA DNA sequence as input, and write GFF3 as
+    output.'
+build:
+  number: 0
+package:
+  name: barrnap
+  version: '0.2'
+requirements:
+  build: []
```

Comment:
Yeah, I probably need to change all these recipes to be more relaxed as soon as we agreed on a way forward with the meta-packages.
Btw. is a build-number increase necessary if I remove this line?